apps/ios/SimpleX SE/ShareViewController.swift

```diff
@@ -26,14 +26,7 @@ class ShareViewController: UIHostingController<ShareView> {
 
     override func viewDidLoad() {
         ShareModel.CompletionHandler.isEventLoopEnabled = false
-        if let extensionContext,
-           let item = extensionContext.inputItems.first as? NSExtensionItem {
-            model.completion = {
-                ShareModel.CompletionHandler.isEventLoopEnabled = false
-                extensionContext.completeRequest(returningItems: [item])
-            }
-            Task { await MainActor.run { model.item = item } }
-        }
+        model.setup(context: extensionContext!)
```

Comment:
Moved to shareModel. `extensionContext` is not @Sendable so the ref has to be passed before entering Task